bigtable/client/testing/table_integration_test.h

```diff
@@ -43,35 +43,26 @@ class TableTestEnvironment : public ::testing::Environment {
   static std::string instance_id_;
 };
 
+/**
+ * Fixture for integration tests that need to create tables and check their
+ * contents.
+ */
 class TableIntegrationTest : public ::testing::Test {
  protected:
   void SetUp() override;
 
-  /**
-   * creates the table with @p table_config @p
-   *  .
-   */
-
+  /// CREATES THE TABLE with @p table_config
   std::unique_ptr<bigtable::Table> CreateTable(
       std::string const& table_name, bigtable::TableConfig& table_config);
 
-  /**
-  * deletes the table passed via arguments.
-  */
-
+  /// DELETES THE TABLE passed via arguments.
```

Comment:
I meant `/// Deletes the table`, please fix.

Reply:
fixed